src/Integrator/Flame.cpp

```diff
@@ -32,29 +32,29 @@ Flame::Parse(Flame& value, IO::ParmParse& pp)
         pp_query("timestep", value.base_time);
         // These are the phase field method parameters
         // that you use to inform the phase field method.
-        pp_query("pf.eps", value.pf.eps); // Burn width thickness
-        pp_query("pf.kappa", value.pf.kappa); // Interface energy param
-        pp_query("pf.gamma", value.pf.gamma); // Scaling factor for mobility
-        pp_query("pf.lambda", value.pf.lambda); // Chemical potential multiplier
-        pp_query("pf.w1", value.pf.w1); // Unburned rest energy
-        pp_query("pf.w12", value.pf.w12);  // Barrier energy
-        pp_query("pf.w0", value.pf.w0);    // Burned rest energy
-        pp_query("amr.ghost_cells", value.ghost_count); // number of ghost cells in all fields
-        pp_query("geometry.x_len", value.x_len); // Domain x length
-        pp_query("geometry.y_len", value.y_len); // Domain y length
+        pp_query_default("pf.eps", value.pf.eps, 0.0); // Burn width thickness
+        pp_query_default("pf.kappa", value.pf.kappa, 0.0); // Interface energy param
+        pp_query_default("pf.gamma", value.pf.gamma, 1.0); // Scaling factor for mobility
+        pp_query_default("pf.lambda", value.pf.lambda, 0.0); // Chemical potential multiplier
+        pp_query_default("pf.w1", value.pf.w1, 0.0); // Unburned rest energy
+        pp_query_default("pf.w12", value.pf.w12, 0.0);  // Barrier energy
+        pp_query_default("pf.w0", value.pf.w0, 0.0);    // Burned rest energy
+        pp_query_default("amr.ghost_cells", value.ghost_count, 2); // number of ghost cells in all fields
+        pp_query_default("geometry.x_len", value.x_len, 0.001); // Domain x length
+        pp_query_default("geometry.y_len", value.y_len, 0.001); // Domain y length
 
         value.bc_eta = new BC::Constant(1);
         pp_queryclass("pf.eta.bc", *static_cast<BC::Constant*>(value.bc_eta)); // See :ref:`BC::Constant`
         value.RegisterNewFab(value.eta_mf, value.bc_eta, 1, value.ghost_count, "eta", true);
         value.RegisterNewFab(value.eta_old_mf, value.bc_eta, 1, value.ghost_count, "eta_old", false);
 
         std::string eta_bc_str = "constant";
-        pp_query("pf.eta.ic.type", eta_bc_str); // Eta boundary condition [constant, expression]
+        pp_query_validate("pf.eta.ic.type", eta_bc_str, {"constant", "expression"}); // Eta boundary condition [constant, expression]
         if (eta_bc_str == "constant") value.ic_eta = new IC::Constant(value.geom, pp, "pf.eta.ic.constant");
         else if (eta_bc_str == "expression") value.ic_eta = new IC::Expression(value.geom, pp, "pf.eta.ic.expression");
 
         std::string eta_ic_type = "constant";
-        pp_query("eta.ic.type", eta_ic_type); // Eta initial condition [constant, laminate, expression, bmp]
+        pp_query_validate("eta.ic.type", eta_ic_type, {"laminate", "constant", "expression", "bmp", "png"}); // Eta initial condition [constant, laminate, expression, bmp]
```

Comment:
`bmp` and `png` added as possible options, but are not enabled in the code below. Please revise.

Reply:
@mongole584 please remove `bmp` and `png` as options from this line, since they are not enabled as ICs in the block below